src/decorator/options/ColumnOptions.ts

```diff
@@ -117,6 +118,11 @@ export interface ColumnOptions extends ColumnCommonOptions {
      */
     asExpression?: string;
 
+    /**
+     * Like asExpression, but support all database in typeorm.
```

Comment:
If it's doing exactly as `asExpression` then we should enable it in other drivers(why duplicate functionality for MySQL). Probably this comment should be changed to describe what feature does not what it is similar to.

Reply:
This is just a reference implementation and may need more improvements.